gremlin-go/driver/client.go

```diff
@@ -33,7 +34,11 @@ type ClientSettings struct {
 	Language        language.Tag
 	AuthInfo        *AuthInfo
 	TlsConfig       *tls.Config
-	Session         string
+	// Minimum amount of concurrent active traversals on a connection to trigger creation of a new connection
```

Comment:
Formatting of this looks off. May need to re-run it.

Reply:
Comments seem to make the formatting like this be deemed as correct.